animation_nodes/nodes/mesh/mesh_object_input.py

```diff
@@ -14,6 +15,7 @@ def create(self):
         self.newInput("Boolean", "Use World Space", "useWorldSpace")
         self.newInput("Boolean", "Use Modifiers", "useModifiers", value = False)
         self.newInput("Boolean", "Load UVs", "loadUVs", value = False)
+        self.newInput("Boolean", "Load Vertex Colors", "loadVertexColors", value = False)
```

Comment:
Can you hide both of those inputs by default?

Reply:
Done.